codemagic.yaml

```diff
@@ -0,0 +1,13 @@
+workflows:
+    production:
```

Comment:
It's widely accepted to use 2-space indentation for YAML files

Reply:
welcome to 2 or 4-space holy war club :D